record_indexer/scripts/combine_individual_collection_stage_indices.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+import json
+import os
+import sys
+
+import requests
+
+def main():
+    ''' combine all indices aliased to `rikolti-stg` into one large index '''
+    endpoint = os.environ.get("OPENSEARCH_ENDPOINT")
+    auth = (os.environ.get("OPENSEARCH_USER"), os.environ.get("OPENSEARCH_PASS"))
```

Comment:
I have, in my env.local

`RIKOLTI_ES_ENDPOINT`, `RIKOLTI_ES_USER`, and `RIKOLTI_ES_PASS`, I also see a `settings.get_auth()`

What are these `OPENSEARCH_` values supposed to be?